src/Client/app/(admin)/sidebar/page.tsx

```
@@ -155,14 +163,17 @@ const Sidebar = () => {
                             </div>
                             {!isCollapsed && (
                                 <Link href="/">
-                                    <button onClick={logout}> Log Out </button>
+                                    <button onClick={()=>{logout();
+                                        setLoading(true);
+                                    }}> Log Out </button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)